### PR TITLE
ADD: new vep_custom_args

### PR DESCRIPTION
**Pull request by maxulysse**

### CODE REVIEW

**Comment by maxulysse** [comment]
closes https://github.com/nf-core/sarek/issues/1003

File: nextflow_schema.json [hunk 1]
```diff
@@ -489,13 +496,15 @@
...
                     "type": "string",
+                    "fa_icon": "fas fa-file",
                     "format": "directory-path",
                     "description": "The output directory where the cache will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
                     "fa_icon": "fas fa-folder-open",
                     "hidden": true
                 },
                 "vep_out_format": {
                     "type": "string",
+                    "fa_icon": "fas fa-file",
```
Comment (FriederikeHanssen) [inline nextflow_schema.json:507]:
```suggestion
```

to fix failing tests

Comment (maxulysse) [reply nextflow_schema.json:507]: I don't this the failing tests are due to the fa_icon

Comment (maxulysse) [reply nextflow_schema.json]: I see, I was thinking about the linting failure...

:facepalm: 

Nevermind me.

**Review by FriederikeHanssen — APPROVED** [review]

**Review by asp8200 — APPROVED** [review]
LGMT.  Just add the possibility to add more custom args to the call to VEP.

**Pull request merged**